media/css/firefox/welcome16.scss

```diff
@@ -8,54 +8,74 @@ $image-path: '/media/protocol/img';
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
 .c-main-image {
-    margin: 0 auto 48px;
+    border-radius: 100%;
+    margin: 0 auto $layout-md;
+    padding: $spacing-lg;
 }
 
 .mzp-c-picto-image {
     align-items: center;
     display: flex;
     min-height: 210px;
 
-    svg {
+    picture {
         display: inline-block;
         margin: 0 auto;
+
+        img {
+            width: 100%;
+        }
     }
 }
 
 .c-coupon-container {
     margin: $spacing-xl 0;
-    background-color: #c3c4fd;
+    background-color: #EDEDF0;
     border-radius: $border-radius-md;
     padding: $spacing-xl $spacing-sm;
 
+    .c-coupon-code-wrapper {
+        display: flex;
+        justify-content: center;
+        margin-bottom: $spacing-lg;
+
+        .c-coupon-code {
+            @include text-body-lg;
+            background-color: $color-white;
+            border: 2px $color-black dotted;
+            border-right: none;
+            border-bottom-left-radius: $border-radius-sm;
+            border-top-left-radius: $border-radius-sm;
+            padding: $spacing-xs $spacing-lg;
+
+            &:hover {
+                    cursor: pointer;
```

Comment:
This might be confusing because it makes the code area seem clickable. I think only the button should get a pointer.